llama_stack/providers/remote/vector_io/pgvector/config.py

```diff
@@ -8,6 +8,10 @@
 
 from pydantic import BaseModel, Field
 
+from llama_stack.providers.utils.kvstore.config import (
+    KVStoreConfig,
+    SqliteKVStoreConfig,
```

Comment:
Given that we dont need to use sqlite as a kvstore (see https://github.com/meta-llama/llama-stack/blob/main/llama_stack/templates/postgres-demo) - should we stop adding SqliteKVStoreConfig to every provider?